### DISCUSSION TITLE
Apache Commons Text vulnerability

### DISCUSSION THREAD

**Discussion by dbelden** [nextgenhealthcare/connect#5509] [Q&A] [answered]
I'm not sure if this is the correct forum to ask this question but here goes:

We've been told by Security we need to **immediately** address a vulnerability they've detected in our Mirth installs.

https://nvd.nist.gov/vuln/detail/CVE-2022-42889

It looks like the fix is fairly simple — "Users are recommended to upgrade to Apache Commons Text 1.10.0, which disables the problematic interpolators by default."

Does anyone have any recommendations on how to address this right away?

Thanks,

-- Daryn

**Comment by jonbartels** [comment, marked as answer]
https://github.com/nextgenhealthcare/connect/issues/5474

**Reply by dbelden** [reply]
Thanks Jon I swear I searched! Even searching for CVE-2022-42889 now though only returns my post as the only hit. :(

5474 is exactly what I needed though.

**Reply by jonbartels** [reply]
No worries. Always call out security questions!
